arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedParquetDefinitionLevelReader.java

```diff
@@ -648,16 +675,15 @@ private void setNulls(
   }
 
   private void setNextNValuesInVector(
-      int typeWidth,
       NullabilityHolder nullabilityHolder,
       VectorizedValuesReader valuesReader,
       int bufferIdx,
       FieldVector vector,
-      int numValues) {
+      int numValues,
+      NumericBaseReader reader) {
```

Comment:
Minor since this is private, but should we just make `setNextNValuesInVector` a private member of `NumericBaseReader`. It looks like this would only apply for that particular reader, and then we can remove passing `this` in NumericBaseReader to this method.